html/homepage.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="/css/homepage-styles.css" />
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
+
+    <title>homepage</title>
+  </head>
+  <body>
+    <nav class="header-container">
+      <img src="/images/image_geometry_1.png" alt="geometry image" />
+      <a href="#">
+        <i class="material-icons">menu</i>
+      </a>
+    </nav>
```

Comment:
- [ ] Hi @Adams-Ghub, great effort working on your portfolio mobile-first, but i would kindly suggest you make the container of this section a `header` tag and wrap only the `hamburger icon` inside of the `nav` tag: 👍 

It would be great if you can re-write your header section in this manner and style it accordingly to fit the Figma design

E.G:
```HTML
<header>
 <img src="/images/image_geometry_1.png" alt="geometry-image" />
   <nav class="header-container">
      <a href="#">
        <i class="material-icons">menu</i>
      </a>
    </nav>
</header>
```
With this, you'll be sure be sure to have semantically re-write your code.

>N.B: The container of the section is a `header` tag.
Typically the `header` tag is used as a page header, it contains the site's title or logo (or both) and the main site navigation. [Check this link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) for more info.